src/neo/Network/P2P/LocalNode.cs

```diff
@@ -169,9 +168,6 @@ protected override void OnReceive(object message)
                 case Message msg:
                     BroadcastMessage(msg);
                     break;
-                case Relay relay:
-                    OnRelay(relay.Inventory);
```

Comment:
This will affect neo-node, when it wants to send and relay transaction.

```cs
// https://github.com/neo-project/neo-node/blob/master/neo-cli/CLI/MainService.Network.cs#L143-L145
tx.Witnesses = context.GetWitnesses();
NeoSystem.LocalNode.Tell(new LocalNode.Relay { Inventory = tx });
Console.WriteLine($"Data relay success, the hash is shown as follows:{Environment.NewLine}{tx.Hash}");
```

Should we keep it, or we just adjust neo-node?

Reply:
Adjust it there I think, @Tommo-L